web/content/css/_site.scss

```
@@ -13,3 +13,11 @@
     margin-top: 70px!important;
   }
 }
+
+._abstract {
+  background-color: #f7f7f7;
+  border-left: 4px solid #025d8c;
+  padding: 15px;
+  margin-bottom: 1.25em;
+  border-radius: 4px;
```

I would go with something very simple like decreasing the intensity of the font color:

```suggestion
  color: #555555;
```

---

However, I didn't check it out in a preview. :)